File: docs-examples/example-java/src/main/java/io/micronaut/configuration/openapi/docs/accessors/Person.java
package io.micronaut.configuration.openapi.docs.accessors;

// tag::imports[]
import io.micronaut.core.annotation.AccessorsStyle;
import io.micronaut.core.annotation.Introspected;
// end::imports[]

// tag::clazz[]
@Introspected
@AccessorsStyle(readPrefixes = "", writePrefixes = "") // <1>
class Person {

    private String name;
    private Integer debtValue;
    private Integer totalGoals;

    Person(String name, Integer debtValue, Integer totalGoals) {
        this.name = name;
        this.debtValue = debtValue;
        this.totalGoals = totalGoals;
    }

    public String name() { // <2>
        return name;
    }

    public Integer debtValue() {
        return debtValue;
    }

    public Integer totalGoals() {
        return totalGoals;
    }

    public void name(String name) { // <2>
        this.name = name;
    }

    public void debtValue(Integer debtValue) {
        this.debtValue = debtValue;
    }

    public void totalGoals(Integer totalGoals) {
        this.totalGoals = totalGoals;
    }
}
// end::clazz[]

File: docs-examples/example-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/accessors/Person.kt
package io.micronaut.configuration.openapi.docs.accessors

// tag::imports[]
import io.micronaut.core.annotation.AccessorsStyle
import io.micronaut.core.annotation.Introspected
// end::imports[]

// tag::clazz[]
@Introspected
@AccessorsStyle(readPrefixes = [""], writePrefixes = [""]) // <1>
class Person(private var name: String, private var debtValue: Int, private var totalGoals: Int) {

    fun name(): String { // <2>
        return name
    }

    fun debtValue(): Int {
        return debtValue
    }

    fun totalGoals(): Int {
        return totalGoals
    }

    fun name(name: String) { // <2>
        this.name = name
    }

    fun debtValue(debtValue: Int) {
        this.debtValue = debtValue
    }

    fun totalGoals(totalGoals: Int) {
        this.totalGoals = totalGoals
    }
}
// end::clazz[]
